src/test/kotlin/no/skatteetaten/aurora/databasehotel/PostgresCleaner.kt

```diff
@@ -0,0 +1,115 @@
+package no.skatteetaten.aurora.databasehotel
+
+import com.zaxxer.hikari.HikariDataSource
+import no.skatteetaten.aurora.databasehotel.dao.DataSourceUtils
+import no.skatteetaten.aurora.databasehotel.dao.Schema
+import no.skatteetaten.aurora.databasehotel.dao.postgres.PostgresDatabaseManager
+import no.skatteetaten.aurora.databasehotel.domain.User
+import no.skatteetaten.aurora.databasehotel.service.DatabaseInstance
+import no.skatteetaten.aurora.databasehotel.service.postgres.PostgresJdbcUrlBuilder
+import javax.sql.DataSource
+
+class PostgresCleaner(dataSource: DataSource) {
```

Comment:
Kanskje et litt rart spørsmål, men tenker bare litt høyt her. 
Burde vi hatt en test for denne? Eller er det kanskje ok så lenge den kun blir brukt i tester? Jeg vet ikke hvilke skader den kan gjøre hvis det er en feil her

Reply:
Jeg vet ikke helt hva som ville vært god måte å teste den her på, clean-metodene her er også basert spesifikt på testene de rydder opp etter.
Gitt at funksjonaliteten sletter databaser og roller kan den gjøre en del skade, men det er ikke mulig å bruke det her utenfor test-scope, og test skal ha sin egen database.

Reply:
Jeg tenker det er greit. Tenkte litt høyt slik at du kunne vurdere det, siden du kjenner koden best :) Ser forresten bra ut :)